packages/theming/atlas/src/theme/native/custom-variables.ts

```diff
@@ -582,3 +582,105 @@ export const slider: VariablesSlider = {
         backgroundColor: background.gray
     }
 };
+
+// == Variables to exclude files
+// ## Reduce the amount of files imported from the Atlas_Core module
+// -------------------------------------------------------------------------------------------------------------------//
+// Activity Indicator
+export const excludeActivityIndicatorDefaultClasses = false;
```

Comment:
For web we do the opposite, so we have "include-widget" variables. @cjfhodges What approach do you want?

Reply:
Exclude statements, follow native's approach.